CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # gradle-nunit-plugin changelog
 ## 1.9
+### Added
 * 'labels' attribute is added to NUnit
+* Added support for multiple 'where' clauses for NUnit v3
```

Comment:
I think we should inform "test" is translated in "where" as "test" is already marked as deprecated in NUnit doc.